app/repositories/MessageInboxRepository.scala

```diff
@@ -96,10 +97,20 @@ class MessageInboxRepositoryImpl @Inject()(
           .find(by(ern, uniqueMessageIdentifier))
           .headOption()
     }
+
+  def delete(ern: String, uniqueMessageIdentifier: Long): Future[Boolean] =
+    keepAlive(ern, uniqueMessageIdentifier).flatMap(_ => {
```

Comment:
I wonder if there is any gain to keep alive the record and then delete it straight afterwards

Reply:
Could there be any reason why we would want to mark the message's last updated time before deleting? AFAIK we're logically deleting it, so will it still remain on the backend's DB?

Reply:
I can't see any reason to, nothing is looking at it and as soon as we update the timestamp we delete the record